packages/create-app/templates/default-app/packages/app/src/App.tsx

```diff
@@ -15,6 +15,7 @@ import { Router as DocsRouter } from '@backstage/plugin-techdocs';
 import { Router as ImportComponentRouter } from '@backstage/plugin-catalog-import';
 import { Router as TechRadarRouter } from '@backstage/plugin-tech-radar';
 import { SearchPage as SearchRouter } from '@backstage/plugin-search';
+import { Router as SettingsRouter } from '@backstage/plugin-user-settings';
```

Comment:
changes in create-app template should have a changeset, with some short instructions for people how to apply the corresponding change manually inside their own repo

Reply:
included changeset